oracle/tests/functions/owner.rs

```diff
@@ -0,0 +1,16 @@
+use crate::utils::{abi_calls::owner, test_helpers::setup, Identity};
+use fuels::{prelude::*, tx::Address};
```

Comment:
There is a warning that apparently this prelude is unused. Could you remove it to check?